yellowbrick/text/dispersion.py

```diff
@@ -47,20 +47,24 @@ class DispersionPlot(TextVisualizer):
     ignore_case : boolean, default: False
 	Specify whether input  will be case-sensitive.
         
+    boundary : boolean, default: False
```

Comment:
I guess I don't mind `boundaries` or even just `annotate_docs`? It might be important to specify whether the end or the beginning of the document is being annotated in the comment.

Reply:
I have decided to go with annotate_docs